### PR TITLE
Update bundler-integration.md

### PR DESCRIPTION
**Pull request by xtuc**
Do not merge yet. I need to figure out how to be available on IRC, what Linux based client are you using?

### CODE REVIEW

File: goals/bundler-integration.md [hunk 1]
```diff
@@ -1,26 +1,35 @@
 # Goal: Bundler Integration
 
-One or two sentence summary of this goal.
+It should be very easy to take Rust code and integrate it into your JavaScript project (using the same build pipeline).
```
Comment (sendilkumarn) [inline goals/bundler-integration.md]: It makes it easy to bundle Rust code to JavaScript project and takes care of compiling to and optimizing WebAssembly. To make it easier and faster for developers to get started with WebAssembly.

File: goals/bundler-integration.md [hunk 2]
```diff
@@ -1,26 +1,35 @@
...
 
 ## Owner
 
 **Sven Sauleau**
 
 - GitHub: [xtuc](https://github.com/xtuc)
 - IRC nick: `username`
```
Comment (sendilkumarn) [inline goals/bundler-integration.md]: You can add my name too here

File: goals/bundler-integration.md [hunk 3]
```diff
@@ -1,26 +1,35 @@
...
 ## Details
 
-More details about this goal. What is in scope for the 2018 edition. What comes
-after that.
+Currently the focus is on the Webpack integration, we should be able to use it with Rust tooling (wasm-bindgen, wasmsnip, wasmopt, etc) for the 2018 edition. 
 
-## How to Help
+### Webpack
+
+The integration is usable, a couple of fixes and improvements are in progress.
```
Comment (sendilkumarn) [inline goals/bundler-integration.md]: I would like to have a table or something here, that tells people what they can use this bundler with and what is the status

Some thing like,

Can bundle Rust code into JS - `YES | NO`
Can bundle WASM code into JS  `YES | NO`
DCE
codesplitting 
etc.,

Comment (sendilkumarn) [reply goals/bundler-integration.md]: Also the status of the entire intergration process

All Good | Usable | Usable with known bugs | Blocker | Cannot use | Not supported yet

File: goals/bundler-integration.md (hunk 2 above)
Comment (xtuc) [reply goals/bundler-integration.md]: I added you in a collaborator section https://github.com/rustwasm/team/pull/194/files#diff-99262f8db9973e8fe5a45e25d031d23eR13, is that ok? I think we should keep on owner responsible for the goal?

**Comment by xtuc** [comment]
Oops, I still wanted to update Sendil's comment https://github.com/rustwasm/team/pull/194#discussion_r193974159.

I will send a follow-up PR.

**Pull request merged**